### DISCUSSION TITLE
iOS SDK vs Swift SDK

### DISCUSSION THREAD

**Discussion by lando213** [awslabs/aws-sdk-swift#1079] [Q&A]
Since the SWIFT SDK will not reach GA this year, the team recommended I use the aws-ios-sdk but it says V2 is built on top of and provides only access to the SWIFT sdk. So, should I use an earlier version of the iOS SDK or am I stuck waiting for the SWIFT GA?

**Comment by parthp027** [comment]
@lando213 see if [this](https://github.com/awslabs/aws-sdk-swift/discussions/1776#discussion-7235591) helps.
